src/features/auth/api/useCreateAuthLogin.ts

```diff
@@ -0,0 +1,29 @@
+import { UseMutationOptions, useMutation } from '@tanstack/react-query'
+import { AuthLoginProps, AuthLoginResponse } from '../types/login'
+import { useApiClient } from '~/providers/ApiClientProvider'
+
+export const useCreateAuthLogin = (
+  options?: UseMutationOptions<
+    AuthLoginResponse,
+    unknown,
+    AuthLoginProps,
+    unknown
+  >
+) => {
+  const { axios, api } = useApiClient()
+
+  return useMutation(
+    ['login-auth'],
```

Comment:
```suggestion
    ['login-auth'],
```
payload bisa dijadiin key harusnya, ga semua gpp

Reply:
create mutationya di declare di atas formik untuk kebutuhan onSubmit jadi gadapet value payload, payload baru di pass di mutateAsync setelah key udh kedeclare